modules/registry-operator-configuration-resource-overview-openstack-swift.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring-registry-storage/configuring-registry-storage-openstack-user-infrastructure.adoc
+
+[id="registry-operator-configuration-resource-overview-openstack-swift_{context}"]
+= Image Registry Operator configuration parameters for OpenStack Swift
+
+The following configuration parameters are available for OpenStack Swift
+registry storage.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`authURL`
+|This value is optional.
+
+|`authVersion`
+|This value is optional.
+
+|`container`
+|This value is optional.
+
+|`domain`
+|This value is optional.
+
+|`domainID`
+|This value is optional.
+
+|`tenant`
+|This value is optional.
+
+|`tenantID`
+|This value is optional.
+
+|`regionName`
+|This value is optional.
+
```

Comment:
It seems all options are marked as optional. A swift registry storage is like this, most should be required I suppose.
    swift:
      authURL: https://rhos-d.infra.prod.upshift.rdu2.redhat.com:13000/v3
      authVersion: "3"
      container: wjosp0816d-p6jqc-image-registry-qqmsgorppmjnoraswnjjtnqrbssbuq
      domain: redhat.com
      domainID: ""
      regionName: regionOne
      tenant: openshift-qe-jenkins
      tenantID: 542c6ebd48bf40fa857fc245c7572e30

Reply:
Thank you, I commented this module out so it won't build. I will need to update it and put it back into the build when more information is available.

Reply:
@bmcelvee @wzheng1 I think everything is optional because if values are empty, we infer the data from the rest of the cluster configuration. This is more important for UPI OpenStack clusters, which IIRC aren't supported across the product yet.

Reply:
The assembly which contains this module is commented out for this release. We have the framework in place to update the content and allow it to build once UPI GAs. At that point we can fill in the table and provide a sample config (ideally).